app/src/components/fields/CustomTextField.tsx

```diff
@@ -0,0 +1,32 @@
+import TextField, { TextFieldProps } from '@material-ui/core/TextField';
+import { IProjectCoordinatorForm } from 'features/projects/components/ProjectCoordinatorForm';
+import { useFormikContext } from 'formik';
+import React from 'react';
+
+export interface ICustomTextField {
+  label: string;
+  name: string;
+  other?: TextFieldProps;
+}
+
+const CustomTextField: React.FC<ICustomTextField> = (props) => {
+  const { touched, errors, values, handleChange } = useFormikContext<IProjectCoordinatorForm>();
+  const { name, label, other } = props;
+
+  return (
+    <TextField
+      name={name}
+      label={label}
+      id={name}
+      onChange={handleChange}
+      variant="outlined"
+      value={values[name]}
+      fullWidth={true}
```

Comment:
Do we not have any textfields without full width? Shouldn't this be part of the other prop so we can change it if needed?

Reply:
lets leave it for now :)